easybuild/easyconfigs/s/shapeit/shapeit-2.r837.GLIBCv2.12_linux_x86_64.eb

```diff
@@ -0,0 +1,24 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'shapeit'
+version = '2.r837.GLIBCv2.12'
```

Comment:
Do we want the `.GLIBCv2.12` part in the version, and thus also in the generated module name?

Also, we try to stick to the 'official' software name here, which seems to be '`SHAPEIT`'.
If you prefer lowercase module names, you should look into instructing EasyBuild to use a custom module naming scheme.

I would suggest this:

```python
name = 'SHAEPIT'
version = '2.r837'
...
sources = ['shapeit.v%(version)s.GLIBCv2.12.Linux.static.tgz']
```

Which would result in a module named `SHAPEIT/2.r837`.

Thoughts?